applications/GeoMechanicsApplication/custom_constitutive/linear_elastic_plane_strain_2D_law.h

```diff
@@ -169,45 +96,27 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoLinearElasticPlaneStrain2DLaw
                             Vector& rStressVector,
                             ConstitutiveLaw::Parameters& rValues) override;
 
-    ///@}
-
-private:
-
-    ///@name Static Member Variables
-    ///@{
+    void UpdateInternalDeltaStrainVector(const Vector& r_strain_vector);
+    void UpdateInternalStrainVectorFinalized(const Vector& r_strain_vector);
```

Comment:
Minor style guide thing: these should be rStrainVector

Reply:
done